### ISSUE TITLE
Cawbird loads briefly, then dies

### ISSUE THREAD

**Issue by mfreeman-xtivia** [IBBoard/cawbird#459]
**Describe the bug**
Starting cawbird 1.5 on Linux Mint 20.3 it briefly shows the application window then dies. Running from the console i see this...

```
mfreeman@mhfmint:~$ cawbird

(cawbird:8244): Gdk-CRITICAL **: 08:59:42.240: gdk_window_get_window_type: assertion 'GDK_IS_WINDOW (window)' failed
**
Pango:ERROR:../pango/pango-layout.c:3991:process_line: assertion failed: (result == BREAK_SOME_FIT || result == BREAK_EMPTY_FIT)
Bail out! Pango:ERROR:../pango/pango-layout.c:3991:process_line: assertion failed: (result == BREAK_SOME_FIT || result == BREAK_EMPTY_FIT)
Aborted (core dumped)

mfreeman@mhfmint:~$ 

```

**Comment by IBBoard** [comment]
This appears to be a duplicate of #410. We're waiting on more information because we can't recreate it.
